analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -699,6 +699,8 @@ resource "aws_iam_policy" "s3tables_policy" {
         Action = [
           "s3tables:UpdateTableMetadataLocation",
           "s3tables:GetNamespace",
+          "s3tables:ListTableBuckets",
+          "s3tables:ListNamespaces",
```

Comment:
Add these to the pod role too. When they fix the library, we can then remove these and use the pod role instead.

Reply:
Added.